### PR TITLE
add Vietnam region & Sea route (constants.go)

### PR DESCRIPTION
**Pull request by nggialac**
Platform/Host: 
VN2/vn2.api.riotgames.com

Regional Routing SEA:
sea.api.riotgames.com

Docs was updated: https://developer.riotgames.com/docs/lol

### CODE REVIEW

File: api/constants.go [hunk 1]
```diff
@@ -27,6 +28,7 @@ const (
 	RouteAmericas Route = "americas"
 	RouteAsia     Route = "asia"
 	RouteEurope   Route = "europe"
+	RouteSea      Route = "sea"
```
Comment (KnutZuidema) [inline api/constants.go:31]:
```suggestion
	RouteSEA      Route = "sea"
```

SEA is an acronym on this case (South-East Asia).

File: api/constants.go [hunk 2]
```diff
@@ -17,6 +17,7 @@ const (
 	RegionTurkey            Region = "tr1"
 	RegionRussia            Region = "ru"
 	RegionPBE               Region = "pbe1"
+	RegionVietNam           Region = "vn2"
```
Comment (KnutZuidema) [inline api/constants.go:20]:
```suggestion
	RegionVietnam           Region = "vn2"
```

**Review by KnutZuidema — CHANGES_REQUESTED** [review]
Thanks for the contribution! Looks fine to me, just please adjust the names.

**Comment by nggialac** [comment]
I think that is ok, glad to this useful

**Review by KnutZuidema — APPROVED** [review]
Thank you!

**Pull request merged**